modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -206,9 +207,11 @@ sda.enkf.multisite <- function(settings, obs.mean, obs.cov, Q = NULL, restart=F,
     ###  Taking care of Forecast. Splitting /  Writting / running / reading back###
     ###-------------------------------------------------------------------------###-----  
     #- Check to see if this is the first run or not and what inputs needs to be sent to write.ensemble configs
-    if (t>1){
+    
+    if (t > 1){
       #removing old simulations
-      unlink(list.files(outdir, "*.nc", recursive = T, full.names = T))
```

Comment:
Output `nc` files are named based on the year of the simulation. So in cases like where we assimilate in a time step smaller than annually, SDA will skip the run part because it thinks the output is already there. So I would suggest that even if you really want to get ride of this line, give it a control flag so the user would have a control over it.

Reply:
@para2x @bailsofhay this is a good suggestion.  Didn't think about cases < 1 year in temporal frequency

Reply:
kk. I can make this fix